dts/bindings/sensor/ams,tsl2540.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022 T-Mobile USA, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    TSL2540 series ambient light sensor.  See datasheet at
+    https://ams.com/documents/20143/36005/TSL2540_DS000564_4-00.pdf/39728ac4-098c-9eca-b5ca-61d9c6f3a588
+
+compatible: "ams,tsl2540"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    required: false
+    description: |
+      Identifies the interrupt pin
+
+  glass-attenuation:
+    type: int
+    default: 227205
+    description: |
+      Visible light attenuation.
+      Integer value for a represenation with 5 decimal points.
+      Typical value is 2.27205 from datasheet.
+      Example: 1.2 would be 120000
+
```

Comment:
That's ok. But I think you need to specify in the description wht that default has been selected. See https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules

Rest is perfect. Thanks